tests/testthat/test-facet-.r

```diff
@@ -36,17 +46,32 @@ test_that("as_facets_list() coerces lists", {
   expect_identical(out, exp)
 })
 
-test_that("as_facets_list() errors with empty specs", {
-  expect_error(as_facets_list(list()), "at least one variable to facet by")
-  expect_error(as_facets_list(. ~ .), "at least one variable to facet by")
-  expect_error(as_facets_list(list(. ~ .)), "at least one variable to facet by")
-  expect_error(as_facets_list(list(NULL)), "at least one variable to facet by")
+test_that("wrap_as_facets_list() and grid_as_facets_list() accept empty specs", {
+  expect_equal(wrap_as_facets_list(NULL), quos())
```

Comment:
I usually use `expect_identical()` for checking quosures, this way the environment is checked as well. Can be a little tricky though sometimes.

Reply:
I got it.